React/Views/RCTView.m

```diff
@@ -341,6 +361,136 @@ - (NSString *)accessibilityValue
   }
   return nil;
 }
+#else // [TODO(macOS ISS#2323203)
+- (id)accessibilityValue {
+  if (self.accessibilityRole == NSAccessibilityCheckBoxRole) {
+    for (NSString *state in self.accessibilityState) {
+      id val = self.accessibilityState[state];
+      if (!val) {
+        continue;
+      }
+      if ([state isEqualToString:@"checked"]) {
```

Comment:
Probably worth defining string consts at the top and just referencing the const down here so there aren't as many floating strings in the body of the file

Reply:
I would but in general react-native doesn't defined consts for the strings that are part of the JS API (JSON) and in this file in particular it would be awkward to have consts for only the mac #if/def instances and not for the iOS or shared instances.